WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFragment.kt

```diff
@@ -139,6 +143,7 @@ class QuickStartFragment : Fragment() {
                 .setPositiveButton(getString(R.string.quick_start_button_skip_positive)) { _, _ ->
                     (view as ScrollView).smoothScrollTo(0, 0)
                     viewModel.skipAllTasks()
+                    AnalyticsTracker.track(QUICK_START_LIST_SKIP_ALL_TAPPED)
```

Comment:
We may want to add the `Stat` qualification for `AnalyticsTracker.track(QUICK_START_LIST_SKIP_ALL_TAPPED)` to produce `AnalyticsTracker.track(Stat.QUICK_START_LIST_SKIP_ALL_TAPPED);` for consistency.